src/ngraph/builder/reshape.cpp

```diff
@@ -229,7 +229,7 @@ shared_ptr<Node> builder::opset1::expand_dims(const Output<Node>& value, size_t
     return builder::opset1::reshape(value, output_shape);
 }
 
-shared_ptr<Node> builder::opset1::squeeze(const Output<Node>& value, vector<int64_t> axes)
+shared_ptr<Node> builder::opset1::squeeze(const Output<Node>& value, vector<size_t> axes)
```

Comment:
Why did you change `int64_t` to `size_t`? The int64_t has a defined size, the size_t is depend on OS bits.

Reply:
Since this is just a helper function which is intended to call with already _normalized_ axes. It internally doesn't handle negative axes so in this way I enforce it's correct behavior. This is not an implementation of `v1::Squeeze` operator which you can find here: https://github.com/NervanaSystems/ngraph/blob/aosewski/group_conv_bprop_v1/src/ngraph/op/fused/squeeze.cpp#L35-L92.